electron.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
If you borrowed this from somewhere, it is good if you mention it on the top of the file.